objects/obj_event/Alarm_0.gml

```diff
@@ -39,10 +39,18 @@ if (obj_controller.fest_type="Great Feast"){
     }
     
     
-    if (obj_controller.fest_feature2=1){
-        var boozer_type;boozer_type=1;
-        if (global.chapter_name="Space Wolves") or (obj_ini.progenitor=3) then boozer_type=2;
-        if (global.chapter_name="Blood Angels") or (obj_ini.progenitor=5) then boozer_type=3;
+    if (obj_controller.fest_feature2 == 1){
+        var boozer_type = 1;
+        if (
+            (global.chapter_name="Space Wolves") || (obj_ini.progenitor == PROGENITOR.SPACE_WOLVES)
```

Comment:
> in theory the `progenitor_map()` function was created to get around these annoying having to specify both "space wolves" and PROGENITOR.SPACE_WOLVES so i would think progenotor map needs updating as a part of this work

I disagree with this a bit, because there could be cases where you may want successor chapters of SW to get something, but not SW themselves.
If the function was accepting an argument to specify if you want to count the progenitor chapter as well - yeah, but it doesn't and if it did - what's the point if you can just have separate enums.

Reply:
> well in these instances youd purely use the progenitor enum not the progenitor_map() the map is purely where you want both which in the current instance is wanted. It seems you acctually agreeing with me i think my origional premis was misunderstood

I mean, if it's just a function to reduce line length for conditionals - it could work, I guess.
Either with a separate argument, to include/exclude the progenitor chapter itself from the check, or without.

Either way, I think it should be renamed, as it doesn't check **only** progenitor of a chapter. Something like "progenitor_or_name()".

Reply:
@jhillacre

Reply:
I'm at work, so I won’t be able to make changes right now, but feel free to merge and add a function if you feel it’s necessary.

My main point is that we still need a mapping of chapter enums to string values anyway. Given that, I’m not sure the function adds enough to justify the call overhead—it essentially just turns into a conditional and check. I get that performance isn’t the main concern, but `and`s or `&&`s in if statements feel like a standard approach here.

Reply:
> it essentially just turns into a conditional `and` check.

Yes and I think that's the original idea. It may kinda help to reduce the amount of typing required, I guess.

I can only say that I'm not proficient enough with code to decide if shortening the line by combining two conditions into one function is worth it in terms of readability, typo chance reduction, verbosness reduction, etc.
Nor can I talk about call overhead concerns.

As such, I'll merge this and if later anyone decides that manually typing two conditions is pain, (@OH296, you can do that too), they can revive this function.
Because, I'll be honest - at this exact moment I don't care either way and just don't want to drag this.